examples/main/main.cpp

```diff
@@ -1012,7 +1042,7 @@ int main(int argc, char ** argv) {
             // output to JSON file
             if (params.output_jsn) {
```

Comment:
```suggestion
            if (params.output_jsn || params.output_jsn_full) {
```

Reply:
Made `ojf` imply `output_jsn` in argument parsing instead :)